pkg/asset/machines/libvirt/machines.go

```diff
@@ -78,3 +78,21 @@ func provider(clusterID string, networkInterfaceAddress string, platform *libvir
 		URI:                     platform.URI,
 	}
 }
+
+func providerMemory(platform *libvirt.Platform) int {
+	// previous default for backward compatibility
+	val := 6144
```

Comment:
secondly, to keep the Machines and MachineSets functions themselves backward compatible, make sure when the pool passed to them doesn't set these, use this conditional then.

Reply:
I think I addressed all your comments, or at least that was the intention. Sorry if I missed something: I'm still learning my way in the codebase.